TenantFile/Api.UnitTests/SmsControllerTests.cs

```diff
@@ -12,11 +12,37 @@
 using Twilio.AspNet.Common;
 using System.Collections.Generic;
 using Microsoft.Extensions.Configuration;
+using Google.Cloud.Firestore;
+using Google.Api.Gax;
+using Google.Cloud.Firestore.V1;
+using TenantFile.Api.Models;
+using TenantFile.Api.Extentions;
+using System.Linq;
 
 namespace Api.UnitTests
 {
-    public class SmsControllerTests
-    {
+    public class SmsControllerTests : IDisposable
+    {               
+        Mock<ICloudStorage> cloudStorageMock;
+        Mock<IDocumentDb> firestoreMock;
+        Timestamp startTimestamp;
+        FirestoreDbBuilder tree;
+        FirestoreDb emulatedFirestore;

```

Comment:
Let's leverage https://github.com/moq/moq4 for doing mocks, it's a pretty awesome library.